src/cdownload.cpp

```diff
@@ -212,7 +212,7 @@ void CDownload::startDownloading() {
     if(connection->pSocket.isValid() && connection->pSocket.state() == QAbstractSocket::ConnectedState) {
         socket.connectToHost(connection->pSocket.peerAddress(), connection->pSocket.peerPort()+1, QIODevice::ReadWrite);
         socket.waitForConnected();
-        if(socket.state() == !QAbstractSocket::ConnectedState) {
+        if(socket.state() != QAbstractSocket::ConnectedState) {
```

Comment:
This was to fix an enum to bool warning, pretty sure this was the intention.

Reply:
Absolutely.